docs/docs/integrating-with-other-libraries.md

```diff
@@ -0,0 +1,205 @@
+---
+id: integrating-with-other-libraries
+title: Integrating with other libraries
+permalink: docs/integrating-with-other-libraries.html
+---
+
+## Using jQuery plugins from React
+
+React is unaware of changes made to the DOM outside of React. It determines updates based on its own internal representation, and if those updates are invalidated, React has no way to recover.
+
+This does not mean it is impossible or even necessarily difficult to combine React with other ways of affecting the DOM, you just have to be mindful of what each are doing.
+
+The easiest way to avoid conflicts is to prevent the React component from updating. This can be done explicitly by returning false from [`componentWillUpdate()`](https://facebook.github.io/react/docs/react-component.html#componentwillupdate), or by rendering elements that have no reason to change.
+
+```js
+class SomePlugin extends React.Component {
+  componentDidMount() {
+    this.$el.somePlugin();
+  }
+
+  componentWillUnmount() {
+    // cleanup
```

Comment:
Couldn't remember if `somePlugin('destroy')` was standard or not. That works out well as I wanted to mention the issue in the code without cluttering it with a full explanation.

As to saving the element instead of the wrapped element on the component, I'm not sure. Anyone familiar with jQuery will know the `$variableName` convention, and it is more performant to wrap it once. But jQuery isn't used nearly as much anymore, so is this still safe to assume? Should I go further and use `jQuery(el)` instead of `$(el)`?